### PR TITLE
Update SRG-OS-000478-GPOS-00223 for RHEL9 STIG

### PR DESCRIPTION
**Pull request by marcusburghardt**
#### Description:

Included fixtext, test scenarios and remediation for `enable_dracut_fips_module`

#### Rationale:

RHEL9 STIG

### CODE REVIEW

File: controls/srg_gpos/SRG-OS-000478-GPOS-00223.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ controls:
     the following: to provision digital signatures, to generate cryptographic hashes,
     and to protect unclassified information requiring confidentiality and cryptographic
     protection in accordance with applicable federal laws, Executive Orders, directives,
-    policies, regulations, and standards.'
```
Comment (Mab879) [inline controls/srg_gpos/SRG-OS-000478-GPOS-00223.yml:9]: What was the reasoning for removing the Oxford comma?

**Review by Mab879 — COMMENTED** [review]
One minor nit.

File: controls/srg_gpos/SRG-OS-000478-GPOS-00223.yml (hunk 1 above)
Comment (marcusburghardt) [reply controls/srg_gpos/SRG-OS-000478-GPOS-00223.yml:9]: Hum, in other languages I am aware, it is grammatically incorrect to insert a comma before `and` in the last element of a enumeration. So, it was a natural reaction from my side. : ) However, after your comment, I researched the nuances of the grammar and learned that this comma is actually optional in English, for this case. Thus, I will respect the author's choice. : ) I have to update this PR soon, then I revert this change. Thanks for pointing this.

**Review by Mab879 — APPROVED** [review]

**Pull request merged**